ModelBundle/Migrations/MongoDB/Version20160304170101.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Migrations\MongoDB;
+
+use AntiMattr\MongoDB\Migrations\AbstractMigration;
+use Doctrine\MongoDB\Database;
+
+/**
+ * Class Version20160304170101
+ */
+class Version20160304170101 extends AbstractMigration
+{
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return "Update users group collection";
```

Comment:
I think it could be nice to be more explicit of what is done in this migration. That way, when we'll have a ton of scripts among several Orchestra versions, it will be easier to find a specific migration script.

Can you add what is altered with this script ? One or multiple field ? A collection name ? Something is added or removed ?